apps/frontend/src/components/Footer.tsx

```diff
@@ -0,0 +1,31 @@
+import { Center, Link, Stack, Text } from '@chakra-ui/react';
+import { githubRepo, slackInvite } from '../links';
+import { CompatibleWithStravaIcon } from './CompatibleWithStravaIcon';
+
+export const Footer = () => {
+  return (
+    <Center pb="20vh" px="2">
+      <Stack maxW="600px" color="#999">
+        <Text>
+          dundring.com is an in-browser training application created to control
+          and track your training with a smart bike trainer.
```

Comment:
last part is maybe a bit in the future, but 😅

Reply:
Yeah, we could probably reword it in the future – but I think it works for now 🤠